scripts/test/playback_gif.py

```diff
@@ -0,0 +1,75 @@
+import os
+from pathlib import Path
+import argparse
+import numpy as np
+import cv2
+from PIL import Image
+import imageio
+
+
+parser = argparse.ArgumentParser(description='format')
+parser.add_argument('--dir', type=str, default= 'vid2vid/datasets/sceneNet', help='Dataset directory')
+parser.add_argument('--res', type=str, default= 'vid2vid/results/depth2room_320_8g', help='Result directory')
+parser.add_argument('--num', type=int, default= 1, help='Number of result versions')
+
+
+def depth_norm(img, depth_min = 0.0, depth_max = 5.0):
+
+    img = 255.0*(1.0 - (img - 1000.0*depth_min) / (1000.0*(depth_max - depth_min)))
```

Comment:
could you break this down to one step at a time please?
img_norm = (img - 1000.0*depth_min)/(1000.0*(depth_max - depth_min))
this I suppose returns values in 0-1 range? What happens if depth_max!=np.max(img), wondering if you need a np.clip here ?
img_inv = 255.0*(1 - img_norm)
Wondering if you are inverting the distance/depth map here?
It might be helpful to include doc-string and one line summary of what the function does.

Reply:
cv2.COLORMAP_JET is supposed to show distance from closest (blue) to furthest (red) but it was showing the opposite. So the purpose of the inverse is to correct this. I will add a short summary for each function soon.